applications/hips/values.yaml

```diff
@@ -10,21 +10,21 @@ config:
   # @default -- None, must be set
   gcsProject: ""
 
-  # -- A mapping of dataset names to locations in GCS.
-  # /api/hips/v2/<dataset-name> will serve files out of the corresponding GCS
+  # -- A mapping of release names to locations in GCS.
+  # /api/hips/v2/<release-name> will serve files out of the corresponding GCS
   # bucket.
   # @default -- None, must be set
-  datasets: null
-    # -- Dataset name
+  releases: null
+    # -- Release name
     # dp02:
     #   -- Name of Google Cloud Storage bucket holding the HiPS files
     #   gcsBucket: somebucket
     # dp1:
     #   -- Name of Google Cloud Storage bucket holding the HiPS files
     #   gcsBucket: somebucket
 
-  # The name of the dataset to serve from the v1 endpoint (/api/hips)
-  defaultDatasetName: null
+  # The name of the release to serve from the v1 endpoint (/api/hips)
+  defaultReleaseName: null
```

Comment:
This is missing the documentation marker so the documentation is empty in `README.md`.